redbiom/commands/admin.py

```diff
@@ -45,13 +45,15 @@ def coherency(context):
               help="The filepath to the table to load.")
 @click.option('--context', required=True, type=str,
               help="The name of the context to load into.")
+@click.option('--tag', required=False, type=str, default=None,
+              help="The tag associated to the samples (e.g., preparation ID).")
```

Comment:
artifact ID?

Reply:
Well, it can be whatever, right? In qiita, we use it as artifact id but in the [internal redbiom documentation](https://github.com/biocore/redbiom/blob/master/redbiom/admin.py#L177) it says prep id. If you think it's worth it, I can change both ...

Reply:
meh